sdk/eventhub/test-resources.bicep

```diff
@@ -32,6 +32,7 @@ resource eventHubNamespace 'Microsoft.EventHub/namespaces@2024-01-01' = {
     zoneRedundant: false
     isAutoInflateEnabled: false
     maximumThroughputUnits: 0
+    disableLocalAuth: subscription().id != '4d042dc6-fe17-4698-a23f-ec6a8d1e98f4'
```

Comment:
if these tests work and don't need to explicitly test this then we should disable all the time.

Reply:
we need to test connection string scenarios, I just stash them in a kv first to prevent leakage: https://github.com/Azure/azure-sdk-for-js/blob/10d6bc826b04699c78b6379102e97776b9656b19/sdk/eventhub/test-resources.bicep#L179